app/assets/stylesheets/application.scss

```diff
@@ -247,6 +249,7 @@
 }
 
 .button-s {
+    margin-top: 5px;
     align-items: center;
```

Comment:
these properties are repeated for all the above classes right? can you refactor them?

Reply:
Yes @iMagesh

Reply:
I removed them as I do not need those stylings @iMagesh, all stylings for button I added using the "flair batch" design class. So I removed them, the button stylings also can be removed if @SrinandhiniM  didn't use it  in other places @iMagesh